src/footer/FandomGlobalFooter.es6

```diff
@@ -2,17 +2,18 @@ import 'whatwg-fetch';
 import getStrings from '../getStrings.es6';
 import getOrCreateTemplate from '../getOrCreateTemplate.es6';
 import footerTemplate from './templates/footer.handlebars';
-import AttributeHelper from '../AttributeHelper.es6';
+import AttributeHelper from '../helpers/AttributeHelper.es6';
+import SvgHelper from '../helpers/svg/SvgHelper.es6';
 import footerStyle from './style.scss';
 
 // FIXME: Can this be included globally or reduced?
```

Comment:
Yeah. Ideally we could use [purifycss](https://github.com/purifycss/purifycss) to strip out unused design system styles to lower the size of the build. I have this semi-working, but it's not ready to ship. We could also do the same for the design system icons if we change how they're used in the footer.